common/survey/categoryItem.js

```diff
@@ -20,7 +20,7 @@ const newItem = (levelUuid, parentItemUuid = null, props = {}) => ({
   [keys.uuid]: uuidv4(),
   [keys.levelUuid]: levelUuid,
   [keys.parentUuid]: parentItemUuid,
-  [keys.props]: props,
+  [keys.props]: ObjectUtils.clean(props),
```

Comment:
where are you creating an item with null values in props? the issue is there..what do you think?

Reply:
this just avoids to have even empty items in the props, like "descriptions" or "extra"